src/Symfony/Component/TypeInfo/Type/BuiltinType.php

```diff
@@ -44,6 +44,15 @@ public function getTypeIdentifier(): TypeIdentifier
         return $this->typeIdentifier;
     }
 
+    public function isA(TypeIdentifier|string $subject): bool
```

Comment:
Since we will be casting a string as an ObjectType, can't we cast a string as a BuiltinType as well in here ?